### PR TITLE
Add AdminGraphqlAPIID to Metafield

### PR DESCRIPTION
**Pull request by sshaw**
I assume c87c24c is not wanted.

### CODE REVIEW

**Review by sandersjessica — APPROVED** [review]

File: collection_test.go [hunk 1]
```diff
@@ -328,7 +328,6 @@ func TestListProductsWithPagination(t *testing.T) {
 		},
 		PreviousPageOptions: nil,
 	}
-	fmt.Println(fmt.Sprintf("NEXT page OPTIONS=%#v", page.NextPageOptions))
```
Comment (ysurtayev-bold) [inline collection_test.go:331]: Thank you for removing this debug print

File: product_listing_test.go [hunk 2]
```diff
@@ -262,7 +262,6 @@ func TestProductListingPublish(t *testing.T) {
 	}
 
 	returnedProduct, err := client.ProductListing.Publish(product.ID)
-	fmt.Printf("%+v", returnedProduct)
```
Comment (ysurtayev-bold) [inline product_listing_test.go:265]: Thank you for removing this debug printf

**Review by ysurtayev-bold — APPROVED** [review]

**Comment by ysurtayev-bold** [comment]
@gordcurrie this one is ready to be merged

**Comment by gordcurrie** [comment]
Thank you for your contribution, there are a few PRs queued up we'll do a release when they're ready to go.

**Pull request merged**